src/Microsoft.AspNet.Session/DistributedSession.cs

```diff
@@ -62,13 +62,13 @@ public void Set(string key, [NotNull] byte[] value)
             if (encodedKey.KeyBytes.Length > KeyLengthLimit)
             {
                 throw new ArgumentOutOfRangeException(nameof(key),
-                    string.Format("The key cannot be longer than '{0}' when encoded with UTF-8.", KeyLengthLimit));
+                    string.Format(Resources.Exception_KeyLengthIsExceeded, KeyLengthLimit));
```

Comment:
There's an overload that does not require string.Format. See Resources.FormatException_KeyLengthIsExceeded